server/conf/i18n/messages

```diff
@@ -503,6 +503,7 @@ validation.numberRequired=Must contain only numbers.
 
 validation.tooFewSelections=Please select at least {0}.
 validation.tooManySelections=Please select fewer than {0}.
+validation.tooManySelections.v2=Please select at most {0}.
```

Comment:
**suggestion:** Could you include a context string?

Reply:
Thanks done!